FreeRTOS/FreeRTOS
# Smart-EVB Support Freertos for Alibaba's risc-v cpu (XUANTIE-E906)

Smart-EVB Support Freertos for Alibaba's risc-v cpu (XUANTIE-E906)

Description
-----------

1, Support Alibaba T-HEAD's Development Board (SMART-EVB) 
2, Add bsp code, include core, driver, libs, example code.
3, Add CDS (Alibaba T-HEAD's IDE) project file.

Test Steps
-----------

Ubuntu Host:
Quick Start for qemu run:
1, toolchain download: 
link: https://occ.t-head.cn/community/download?id=3913221581316624384
version: riscv64-elf-x86_64-20210512.tar.gz
2, qemu download
link: https://occ.t-head.cn/community/download?id=636946310057951232
version: csky-qemu-x86_64-Ubuntu-16.04-20210202-1445.tar
3, cd ../FreeRTOS/Demo/RISC-V_RV32_THEAD_SMART_CDS/projects/task/freertos
4, make
5, qemu-system-riscv32 -cpu e906 -M smartl -kernel out/smartl_e906_evb.elf -nographic

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
